Content.Shared/Movement/Pulling/Systems/PullingSystem.cs

```diff
@@ -823,7 +822,10 @@ public bool TryGrab(Entity<PullableComponent?> pullable, Entity<PullerComponent?
         if (!ignoreCombatMode)
             if (!_combatMode.IsInCombatMode(puller))
                 return false;
-
+        // Corvax-Next-GrabComponent-Start
+            if (!HasComp<CanChokeGrabComponent>(puller))
+                return false;
+        // Corvax-Next-GrabComponent-End
```

Comment:
Если тут нет логической ошибки, то есть ошибка в рефакторе:

```suggestion

        // Corvax-Next-GrabComponent-Start
        if (!HasComp<CanChokeGrabComponent>(puller))
            return false;
        // Corvax-Next-GrabComponent-End
        
```

Reply:
И я бы на просто Grab поменял бы.